install-template.sh

```diff
@@ -28,11 +34,23 @@ if test_nix_installation; then
   echo Hooray, nix is already installed:
   nix --version
 else
-  echo \'@@toolName@@\' depends on nix, but it seems that you don\'t have a nix installation.
-  echo This installer will run the default nix installer \(version 2.17.1\) for you.
-  echo That installer will ask for your admin password to install nix.
-  echo For more information, see: https://nixos.org/download
-  echo
+  cat << EOF
+Welcome to the '@@toolName@@' installer!
+
+'@@toolName@@' depends on nix, but it seems that you don't have a nix
+installation. This installer will install nix for you. It uses the
+'nix-installer' by Determinate Systems. For more information, see
+https://github.com/DeterminateSystems/nix-installer. That installer
+will ask for your admin password to install nix.
```

Comment:
This last sentence is confusing: “that” seems to refer to “something else”.

Maybe have the sudo comment as a separate paragraph, and say “This” instead?